test/fixtures/multiplatform/bwb.xcodeproj/xcshareddata/xcschemes/Lib (watchOS) (47488).xcscheme

```diff
@@ -14,9 +14,9 @@
                <EnvironmentBuildable>
                   <BuildableReference
                      BuildableIdentifier = "primary"
-                     BlueprintIdentifier = "83D00AD7532094902D925C83"
-                     BuildableName = "Lib (watchOS)"
-                     BlueprintName = "Lib (watchOS)"
+                     BlueprintIdentifier = "D1F5DE7EC1DF3772DDE52967"
+                     BuildableName = "Lib (watchOS) (47488)"
```

Comment:
Seems that the test version of the watch app or watch extension is being transitioned differently than the one being embedded into the app. I'll dig into this for a bit to see what's happening.

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/pull/991#discussion_r958884178